consensus/state_sync/helpers.go

```diff
@@ -34,13 +34,13 @@ func (m *stateSync) sendToPeer(msg *anypb.Any, peerId cryptoPocket.Address) erro
 func getMessageType(msg *typesCons.StateSyncMessage) string {
 	switch msg.Message.(type) {
 	case *typesCons.StateSyncMessage_MetadataReq:
-		return "MetadataRequest"
+		return "PeerMetadataRequest"
 	case *typesCons.StateSyncMessage_MetadataRes:
-		return "MetadataResponse"
+		return "PeerMetadataResponse"
```

Comment:
Let's be consistent with `StateSyncMetadataRequest` or `MetadataRequest` etc..

We're mixing things all over. What's your thinking?

Reply:
I think it is best to update all to actual proto names, to remove confusion and mixing, so:
- StateSyncMetadataRequest
- StateSyncMetadataResponse 
- GetBlockRequest 
- GetBlockResponse

Wdyt?

Reply:
Good idea, let's do it